packages/plugin-ext/src/main/browser/view/tree-view-widget.tsx

```diff
@@ -281,7 +281,8 @@ export class TreeViewWidget extends TreeWidget {
             });
         }
 
-        nodes.push(<div>{work}</div>);
+        const classes = TREE_NODE_SEGMENT_CLASS + ' ' + TREE_NODE_SEGMENT_GROW_CLASS;
+        nodes.push(<div className={classes}> {work}</div >);
```

Comment:
[minor] you can simply do:

```ts
const classNames = [TREE_NODE_SEGMENT_CLASS, TREE_NODE_SEGMENT_GROW_CLASS].join(' ');
nodes.push(<div className={classNames}>{work}</div >);
```

Which allows us to  add classes to `classes`  more easily.
It is also done in many areas in the code, example:

https://github.com/eclipse-theia/theia/blob/9fb6351621583f4a3e51954debbea9ed1e080168/packages/plugin-ext/src/main/browser/view/tree-view-widget.tsx#L316